vue/mixins/device.js

```diff
@@ -0,0 +1,49 @@
+const DESKTOP_CLASS = "desktop";
+const TABEL_CLASS = "tablet";
+const MOBILE_CLASS = "mobile";
+
+const TABLET_WIDTH = 768;
+const MOBILE_WIDTH = 420;
+
+const deviceMixin = {
+    methods: {
+        setTabletWidth(width, update = false) {
+            this.tabletWidth = width;
+            if (update) this.updateDeviceClass();
+        },
+        setMobileWidth(width, update = false) {
+            this.mobileWidth = width;
+            if (update) this.updateDeviceClass();
+        },
+        listenDeviceChange() {
+            window.addEventListener("resize", () => this.updateDeviceClass());
+        },
+        updateDeviceClass() {
+            const width = window.innerWidth;

```

Comment:
I've seen this code elsewhere... where was it?

Reply:
https://github.com/ripe-tech/ripe-commons-pluginus/blob/master/js/util/device.js